src/models/User/index.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+import createUser from './helpers/createUser'
+export * from './helpers/createUser'
```

Comment:
To export the type as well, so that you don’t have to `import type { CreateUserArgs } from '@models/User/helpers/createUser`

Reply:
The import system is kinda weird since Babel 6 import/export modules implements the ES modules spec, but we’ll get used to it.